src/lib/config/mainnet/pools.ts

```diff
@@ -253,6 +253,8 @@ const pools: Pools = {
       '0xd689abc77b82803f22c49de5c8a0049cc74d11fd000200000000000000000524',
       '0x42fbd9f666aacc0026ca1b88c94259519e03dd67000200000000000000000507',
       '0x3e953c6bf97284f736c5f95b3c3be8f4e48075f4000200000000000000000522',
+      '0x793f2d5cd52dfafe7a1a1b0b3988940ba2d6a63d0000000000000000000004f8',
+      '0x7e9afd25f5ec0eb24d7d4b089ae7ecb9651c8b1f000000000000000000000511',
```

Comment:
This pool won't load for me in testing. It's not in the graph yet so it's falling back to loading using on-chain data, and thenit's saying `totalSupply` is null (though etherscan returns 0 for `getActualSupply` here https://etherscan.io/address/0x7e9afd25f5ec0eb24d7d4b089ae7ecb9651c8b1f#readContract).